package tests.exception;

// to do: use assertions.assertThrow to check what exception is thrown
import static org.junit.jupiter.api.Assertions.assertTrue;

import org.junit.jupiter.api.Test;



public class MeetingExceptionTest {

	@Test
	public void testAddMeetingException() {
		assertTrue(true);
	}
}
